package.json

```diff
@@ -98,5 +98,14 @@
   "packageManager": "yarn@1.22.19",
   "engines": {
     "node": ">=20.0.0"
+  },
+  "resolutions": {
```

Comment:
Its not the ideal, but after removing the package libsignal-node im getting error in `yarn lint:fix`
@canove Do you know what I can do?